sbin/build.sh

```diff
@@ -968,17 +946,17 @@ cleanAndMoveArchiveFiles() {
 
   echo "Currently at '${PWD}'"
 
-  local jdkPath=$(ls -d ${BUILD_CONFIG[JDK_PATH]})
+  local jdkPath=$(ls -d "${BUILD_CONFIG[JDK_PATH]}")
```

Comment:
This line needs the quotes removing as it relies on glob expansion.
Please can you undo the other " changes above as well